examples/with-antd/project_options.json

```diff
@@ -4,14 +4,17 @@
   "config": {
     "entry": [
       {
-        "import": "./src/index.tsx",
-        "library": {
```

Comment:
the `apps` config is default here?

Reply:
No, library diffs from app only at the point at whether there is a library config in entry options, see https://github.com/webpack/webpack/blob/main/declarations/WebpackOptions.d.ts#L1189, this is an optional config, when it is set, it's a library, when it's not, it's an app.That is very clear, no need to explain be example.

Reply:
We filter the apps and librarys like this
<img width="722" alt="image" src="https://github.com/user-attachments/assets/0b729208-cd1a-4080-98e0-a1a7655fda67" />

Reply:
get~